controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorGroupImpl.java

```diff
@@ -82,7 +82,13 @@
 
     void initialize() throws CheckpointStoreException {
```

Comment:
If I understand it correctly, initialize can be called multiple times, which causes the problem with adding reader groups. If this is right, then we need to make sure that initialize is idempotent. Would it make sense to add a test case that invokes initialize twice and make sure that nothing is thrown?